README.md

```diff
@@ -1,6 +1,17 @@
 # eslint-plugin-expect-type
 
-ESLint plugin with $ExpectType and $ExpectError type assertions
+> ESLint plugin with `^?` Twoslash, `$ExpectError`, and `$ExpectType` type assertions. ✨
```

Comment:
Can be in a separate PR, but it would be helpful to include some background on why you might want to do type assertions (testing types) and specifically why you might want to do them _in this way_ (string matching avoids vagaries of assignability and lets you test the display of types).